pkg/dependency/credhelper/cred_helper_binary.go

```diff
@@ -0,0 +1,214 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package credhelper
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/spf13/afero"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/config"
+	"github.com/runfinch/finch/pkg/dependency"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/path"
+)
+
+// ConfigFile is taken from https://github.com/docker/cli/blob/master/cli/config/configfile/file.go
+type ConfigFile struct {
```

Comment:
Can we just import this? Something like (at the top):

```go
"https://github.com/docker/cli/blob/master/cli/config/configfile"
```

And then when you use the struct:

```go
configfile.ConfigFile
```